spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/AdminConfiguration.java

```diff
@@ -86,7 +86,12 @@ public ModuleRegistry moduleRegistry() {
 	protected static class LocalConfig {
 
 		@Bean
-		public ModuleDeployer moduleDeployer(ModuleLauncher moduleLauncher) {
+		public ModuleDeployer longRunningModuleDeployer(ModuleLauncher moduleLauncher) {
```

Comment:
Since we are creating two same beans with different names for the purpose of using them in qualifiers, can we do this way: 
```
@Bean(name={"longRunningModuleDeployer","taskModuleDeployer"})
```